src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -464,14 +599,141 @@ struct ConvertPrecision<std::tuple<ov::float16, ov::float16>> {
+        const size_t iterations = ov::intel_cpu::div_up(ctx.size, batch);
+        auto clamp = ctx.clamp;
+
+        if (!clamp) {
+            parallel_for(iterations, [&](size_t i) {
+                const size_t offset = i * batch;
+                const size_t current_batch_size = std::min(ctx.size - offset, batch);
+                jit_convert(src + offset, dst + offset, current_batch_size);  // fp32 -> fp16
+            });
+        } else {
+            parallel_for(iterations, [&](size_t i) {
+                const size_t offset = i * batch;
+                const size_t current_batch_size = std::min(ctx.size - offset, batch);
+                jit_convert<float, ov::float16, true>(src + offset, dst + offset, current_batch_size);  // fp32 -> fp16
+            });
+        }
+
+        ctx.converted = true;
+    }
+};
+
+template <>
+struct ConvertPrecision<std::tuple<uint8_t, ov::float16>> {
+    void operator()(ConvertContext& ctx) {
+        auto src = static_cast<const uint8_t*>(ctx.srcPtr);
+        auto dst = static_cast<ov::float16*>(ctx.dstPtr);
+
+        constexpr size_t batch = 64;
+        const size_t iterations = ov::intel_cpu::div_up(ctx.size, batch);
+        auto clamp = ctx.clamp;
+
+        if (!clamp) {
+            parallel_for(iterations, [&](size_t i) {
+                const size_t offset = i * batch;
+                const size_t current_batch_size = std::min(ctx.size - offset, batch);
+                jit_convert(src + offset, dst + offset, current_batch_size);  // src_t -> fp16
+            });
+
+        } else {
+            typedef float batch_type[batch];
+            uint8_t lbound, ubound;
+            std::tie(lbound, ubound) = ctx.range<uint8_t>();
+            parallel_for(iterations, [&](size_t i) {
+                batch_type tmp;
+                const size_t offset = i * batch;
+                const size_t current_batch_size = std::min(ctx.size - offset, batch);
+                for (size_t j = 0; j < current_batch_size; ++j)  // src_t -> fp32
+                    tmp[j] = static_cast<float>(std::max(std::min(src[offset + j], ubound), lbound));
+                jit_convert(tmp, dst + offset, current_batch_size);  // fp32 -> fp16
+            });
+        }
+
+        ctx.converted = true;
+    }
+};
+
+template <>
+struct ConvertPrecision<std::tuple<float, int8_t>> {
+    void operator()(ConvertContext& ctx) {
+        auto src = static_cast<const float*>(ctx.srcPtr);
+        auto dst = static_cast<int8_t*>(ctx.dstPtr);
+
+        constexpr size_t batch = 64;
+        const size_t iterations = ov::intel_cpu::div_up(ctx.size, batch);
+        auto clamp = ctx.clamp;
+
+        if (!clamp) {
+            parallel_for(iterations, [&](size_t i) {
+                const size_t offset = i * batch;
+                const size_t current_batch_size = std::min(ctx.size - offset, batch);
+                jit_convert(src + offset, dst + offset, current_batch_size);  // fp32 -> fp16
```

Comment:
Please update the description to be
// fp32 ->int8

Reply:
Done